en/02_Developer_Guides/03_Forms/01_Validation.md

```diff
@@ -6,6 +6,10 @@ icon: check-square
 
 # Form validation
 
+> [!HINT]
+> Before you start implementing custom validation logic, check out [validation using `symfony/validator` constraints](/developer_guides/model/validation/#validation-and-constraints)
+> and see if there's an existing constraint that can do the heavy lifting for you.
```

Comment:
Using the new style here because I don't know if this will be merged before or after the conversion takes place - and since this branch won't be published on the site until April it makes more sense to do this than the old style.